.github/workflows/github-actions-build.yml

```diff
@@ -47,6 +47,10 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: lts/*
+
+      - name: Remove Nodejs
+        run: |
+          sudo apt-get remove nodejs
```

Comment:
any reasons for removing this?

Reply:
Yes , Earlier our github actions build was failing because of some issue with node js package manager npm with ubuntu version we pull. But seems this issue is no longer happening and builds again have started to pass. So this change is no longer needed now and the code changes has been removed.